SCClassLibrary/External/Ableton/SCClockMeterSync.sc

```diff
@@ -0,0 +1,246 @@
+// originally conceived for LinkClock (Ableton Link support)
+// but this may in theory apply to any clock that answers to
+// baseBarBeat_ (e.g. MIDISyncClocks, or BeaconClock)
+
+// TODO OSC message prefix with SC_
+
+SCClockMeterSync {
+	var <clock, <>id, <ports;
```

Comment:
i don't think `id` should be settable once this object is constructed -- otherwise you leave open the possibility for it to be changed while messages are in flight. same with `ports`. it's probably simpler (i.e., less error prone) in those cases to just require users to construct a new object.

Reply:
OK.